Sources/tbDEX/Protocol/Models/Message.swift

```diff
@@ -120,5 +130,7 @@ public struct MessageMetadata: Codable, Equatable {
         case to
         case exchangeID = "exchangeId"
         case createdAt
+        case externalID = "externalId"
+        case `protocol`
```

Comment:
just to double check my swift knowledge - "protocol" is written as `protocol` coz it's a reserved keyword in swift, yea?